packages/fxa-auth-server/lib/redis.js

```diff
@@ -111,6 +111,9 @@ class FxaRedis {
    * @param {AccessToken} token
    */
   setAccessToken(token) {
+    if (token.ttl < 1) {
```

Comment:
It is unclear to me why this was added? Is this to avoid storing pocket tokens?

Reply:
it's a guard against setting and invalid ttl. redis would throw an error with a negative ttl. it shouldn't happen in prod, maybe we should log it...